projects/js-packages/publicize-components/src/components/form/index.js

```diff
@@ -22,6 +22,9 @@ import Notice from '../notice';
 import PublicizeSettingsButton from '../settings-button';
 import styles from './styles.module.scss';
 
+const checkConnectionCode = ( connection, code ) =>
+	false === connection.is_healthy && code === ( connection.error_code ?? 'broken' );
```

Comment:
@pablinos we have to keep false here, because is_healthy is only defined after we come back from the connection test call. Otherwise everything becomes broken, until that call is finisehd